docs/usage/shoot_serviceaccounts.md

```diff
@@ -53,3 +53,34 @@ It has the following specification:
+
+## Managed Service Account Issuer
+
+### Prerequisites
+
+Gardener also provides a way to manage the service account issuer of a shoot cluster as well as serving its OIDC discovery documents from a centrally managed server called [Gardener Discovery Server](https://github.com/gardener/gardener-discovery-server). This ability removes the need for changing the `.spec.kubernetes.kubeAPIServer.serviceAccountConfig.issuer` and exposing it separately. The feature has the following prerequisites:
+- The Garden Cluster should have the Gardener Discovery Server deployed and configured.
+  The easiest way to handle this is by using the [gardener-operator](../concepts/operator.md#gardener-discovery-server).
+- The [`ShootManagedIssuer`](../deployment/feature_gates.md#list-of-feature-gates) feature gate should be enabled.
+
+If uncertain that these prerequisites are met, please contact your Gardener Administrator.
+
+### Enablement
+
+If the prerequisites are met then the feature can be enabled for a shoot cluster by annotating it with `authentication.gardener.cloud/issuer=managed`. Mind that once enabled this feature cannot be disabled. After the shoot is reconciled you can retrieve the new shoot service account issuer value from the shoot's status. A sample query that will retrieve the managed issuer looks like this:
+
+```bash
+kubectl -n my-project get shoot my-shoot -o jsonpath='{.status.advertisedAddresses[?(@.name=="service-account-issuer")].url}'
+```
+
+Once retrieved the shoot's OIDC discovery documents can be explored by querying the `/.well-known/openid-configuration` endpoint of the issuer.
+
+Mind that this annotation is incompatible with the `.spec.kubernetes.kubeAPIServer.serviceAccountConfig.issuer` field so if you want to enable it then the `issuer` field should not be set in the shoot specification.
+
+⚠️ Caution:  If you change from the default issuer to a managed issuer, all previously issued tokens will still be valid/accepted.
+However, if you change from a custom `issuer` `A` to a managed issuer, then you have to add `A` to the `acceptedIssuers` so that previously issued tokens are not invalidated.
+Otherwise, the control plane components as well as system components and your workload pods might fail.
+You can remove `A` from the `acceptedIssuers` when all currently active tokens have been issued solely by the managed issuer.
+This can be ensured by using [projected token volumes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) with a short validity, or by rolling out all pods.
+Additionally, all [`ServiceAccount` token secrets](https://kubernetes.io/docs/concepts/configuration/secret/#service-account-token-secrets) should be recreated.
+Apart from this, you should wait for at least `12h` to make sure the control plane and system components have received a new token from Gardener.
```

Comment:
```suggestion
> [!CAUTION]
> If you change from the default issuer to a managed issuer, all previously issued tokens will still be valid/accepted.
> However, if you change from a custom `issuer` `A` to a managed issuer, then you have to add `A` to the `acceptedIssuers` so that previously issued tokens are not invalidated.
> Otherwise, the control plane components as well as system components and your workload pods might fail.
> You can remove `A` from the `acceptedIssuers` when all currently active tokens have been issued solely by the managed issuer.
> This can be ensured by using [projected token volumes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) with a short validity, or by rolling out all pods.
> Additionally, all [`ServiceAccount` token secrets](https://kubernetes.io/docs/concepts/configuration/secret/#service-account-token-secrets) should be recreated.
> Apart from this, you should wait for at least `12h` to make sure the control plane and system components have received a new token from Gardener.
```